docs/source/api/python/data.rst

```diff
@@ -57,3 +57,14 @@ Protein-Protein Interaction dataset
 
 .. autoclass:: PPIDataset
     :members: __getitem__, __len__
+
+Molecular Graphs
+````````````````
+
+To work on molecular graphs, make sure you have installed `RDKit <https://www.rdkit.org/docs/Install.html>`__.
```

Comment:
Add suggested version here also.

Reply:
Seems that you commented at one place twice?